index.html

```diff
@@ -671,10 +671,241 @@
 				<p class="ednote">Placeholder for inter-publication search.</p>
 			</section>
 
-			<section id="enh-pagination">
-				<h3>Pagination</h3>
+			<section id="Layout">
+				<h3>Layout</h3>
+
+				<p class="issue advisement">This section needs thorough review not only from the publishing community
+					to ensure that the proposed behavior is desirable,
+					but also from Web Browser vendors and other implementors of layout engines
+					to ensure that is it implementable.</p>
+
+				<p>The layout and rendering of <a>Web Publications</a> is governed by the usual rules that apply to all web content.
+				[[!HTML]] (including XHTML) documents are styled and laid out according to the rules of <a href="https://www.w3.org/TR/CSS/">CSS</a>,
+				[[!SVG]] files are rendered as normally expected of that format, and so on.
+				This specification requires no particular profile or subset of CSS, HTML, or SVG to be supported,
+				other than the expectations set for these technologies by their respective specifications.</p>
+
+				<div class=note>
+					<p>This specification intentionally refrains from introducing any new layout feature.
+					Any shortcoming of the web platform in terms of layout needs to be addressed for the whole web platform,
+					which means via CSS.</p>
+
+					<p>This working group recognizes that the features offered by CSS specifications have limitations,
+					and that in some cases implementations may be incomplete.
+					However, attempts to address these shortcomings for the exclusive purpose of <a>Web Publications</a>
+					would at best have no effect,
+					and at worst lead to a divergence between “ordinary” web content and <a>Web Publications</a>,
+					instead of the convergence this specification aims to achieve.</p>
+
+					<p>This working group will work with other relevant groups of the W3C to address
+					platform wide limitations that negatively impacts <a>Web Publications</a>,
+					and invites readers to do the same.</p>
+				</div>
+
+				<p>For the purposes of layout, each resource of a <a>Web Publication</a> is treated as a separate document.
+				User Agents must not mix content from multiple resources in the same rendering:
+				For instance, <a href="https://www.w3.org/TR/CSS21/visuren.html#floats">CSS floats</a> or <a href="https://www.w3.org/TR/CSS21/visuren.html#absolutely-positioned">Absolutely positioned elements</a>
+				from one resource may not intrude or overlap with content from an other resource.</p>
+
+				<p>Users must therefore rely on the UI features of the User Agent
+				to navigate from one resource to another,
```

Comment:
A publication may present that navigation not the UA itself

Reply:
Are you saying that UAs should not provide a UI for navigating between the resources?

We should have an API to let publication provide additional or alternative means of dealing with this, but by default I'd expect the UA to provide some means of navigating the table of contents, switching chapters, and so on, otherwise I am not sure in what sense it would be web publication aware.